bin/functions

```diff
@@ -14,4 +14,9 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+if (process.version !== 'v6.9.1') {
```

Comment:
I don't think it will be a big deal to maintain:

* How many times has the Node.js version used by Google Cloud Functions actually changed?
* We have you and me to help keep this in sync.
* Finally, I could devise a system test that verifies that this library checks for the same Node.js version that's used by the production Cloud Functions service. This test would probably take a couple minutes to run. With this test, the versions would never get out of sync.

Why I think this should be here:

The emulator needs to mimic the Google Cloud Functions environment as closely as possible. If the environment changes, we can change the emulator.